tests/e2e/initialization/chain/main.go

```diff
@@ -50,7 +50,7 @@ func main() {
 
 	b, _ := json.Marshal(createdChain)
 	fileName := fmt.Sprintf("%v/%v-encode", dataDir, chainId)
-	if err = os.WriteFile(fileName, b, 0o777); err != nil {
+	if err = os.WriteFile(fileName, b, 0o777); err != nil { //nolint:gosec
```

Comment:
Does this need to be 777? typically that's hacky instead of just doing 700 or so.

Reply:
If it passes with 600 then that's fine